src/index.html

```diff
@@ -6,8 +6,49 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
     <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="page" data-qa="card">
+      <div class="product-card">
+        <img
+          class="product-card-image"
+          src="../src/images/imac.jpeg"
+          alt="image"
+        >
+        <span class="product-card-text">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </span>
+
+        <span class="product-card-code">Product code: 195434</span>
```

Comment:
```suggestion
        <p class="product-card-code">Product code: 195434</p>
```
Consider leveraging semantic tags